src/Data/Str.php

```diff
@@ -171,29 +171,107 @@ public static function ConvertCase(string $str, $encoding = null)
     }
 
     /**
-     * Checks if given string is a valid Base64 encoded string.
+     * Check if the input string is valid base64.
      *
-     * @param string $str String to be tested
+     * @param string $string String to be tested
      *
-     * @return bool wether it is a valid base64 encoded string or not
+     * @return bool
+     */
+    public static function isBase64(string $string)
+    {
+        // Check if there are valid base64 characters
+        if (!preg_match('/^[a-zA-Z0-9\/\r\n+]*={0,2}$/', $string)) {
```

Comment:
As far as I know, the `preg_match` function will return `integer` value.

I think it can consider using the `preg_match('/^[a-zA-Z0-9\/\r\n+]*={0,2}$/', $string) === 0` condition instead.

Reply:
preg_match() returns either 0 or a bool. 0 and 1 are both considered boolean, I'll just follow your recommendation and do make a strict conditional.